apps/meteor/client/views/room/HeaderV2/ParentRoom/ParentRoom.tsx

```diff
@@ -0,0 +1,22 @@
+import type { IRoom } from '@rocket.chat/core-typings';
+
+import ParentDiscussion from './ParentDiscussion';
+import ParentTeam from './ParentTeam';
+
+const ParentRoom = ({ room }: { room: IRoom }) => {
+	const parentRoomId = room.prid || (room.teamId && !room.teamMain);
```

Comment:
this looks more like a boolean, right?
seems odd for it to be either an id (`room.prid`) OR a boolean that indicates if it's a channel from a team.
i'd suggest 
```suggestion
	const hasParentRoom = Boolean(room.prid) || (room.teamId && !room.teamMain);
```